tests/e2e/docker/run_e2e_single_az.sh

```diff
@@ -25,6 +25,21 @@ if [ -z ${AWS_AVAILABILITY_ZONES}  ];then
 	exit 1
 fi
 
+if [ -z ${OSC_ACCESS_KEY} ]; then
+	echo "OSC_ACCESS_KEY is mandatory to make test pass"
+	exit 1
+fi
+
+if [ -z ${OSC_SECRET_KEY} ]; then
+	echo "OSC_SECRET_KEY is mandatory to make test pass"
+	exit 1
+fi
+
+if [ -z ${OSC_REGION} ]; then
+	echo "OSC_REGION is mandatory to make test pass"
+	exit 1
+fi
+
```

Comment:
just a suggestion: you can keep a `exit_code` variable and exit after all checks (like [this](https://github.com/outscale/osc-sdk-go/blob/v2/.github/scripts/check-creds-settings.sh)). This allow to show all env variable errors at once, this help to fix them quicker.